<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>815</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>720</width>
    <height>815</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>720</width>
    <height>815</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>608</x>
     <y>769</y>
     <width>83</width>
     <height>37</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>1</x>
     <y>492</y>
     <width>706</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>5</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Channel Archiver</string>
    </attribute>
    <widget class="QComboBox" name="cbChannel" >
     <property name="geometry" >
      <rect>
       <x>150</x>
       <y>89</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>TMS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>VMS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>CLS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>HRS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>HDS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>QDS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ICRF ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ECH ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>FUEL ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>CCS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>MPS ArchiveEngine</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ntpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>sysMonitor</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>7</x>
       <y>84</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Service Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="startChannel" >
     <property name="geometry" >
      <rect>
       <x>359</x>
       <y>89</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopChannel" >
     <property name="geometry" >
      <rect>
       <x>457</x>
       <y>89</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>MDSPlus</string>
    </attribute>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>7</x>
       <y>82</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Service Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="stopMdsplus" >
     <property name="geometry" >
      <rect>
       <x>457</x>
       <y>87</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMdsplus" >
     <property name="geometry" >
      <rect>
       <x>150</x>
       <y>87</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>mdsip</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ntpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>sysMonitor</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="startMdsplus" >
     <property name="geometry" >
      <rect>
       <x>359</x>
       <y>87</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>DataAnalysis</string>
    </attribute>
    <widget class="QPushButton" name="startData" >
     <property name="geometry" >
      <rect>
       <x>359</x>
       <y>89</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopData" >
     <property name="geometry" >
      <rect>
       <x>457</x>
       <y>89</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>7</x>
       <y>84</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Service Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbData" >
     <property name="geometry" >
      <rect>
       <x>150</x>
       <y>89</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>ntpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>sysMonitor</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>httpd</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4" >
    <attribute name="title" >
     <string>Backup</string>
    </attribute>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>11</x>
       <y>84</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Service Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbBackup" >
     <property name="geometry" >
      <rect>
       <x>154</x>
       <y>89</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>nsrd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ntpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>sysMonitor</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="startBackup" >
     <property name="geometry" >
      <rect>
       <x>363</x>
       <y>89</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBackup" >
     <property name="geometry" >
      <rect>
       <x>461</x>
       <y>89</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6" >
    <attribute name="title" >
     <string>Web</string>
    </attribute>
    <widget class="QComboBox" name="cbWeb" >
     <property name="geometry" >
      <rect>
       <x>158</x>
       <y>97</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>ntpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>sysMonitor</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>httpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>tomcat</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="startWeb" >
     <property name="geometry" >
      <rect>
       <x>367</x>
       <y>97</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10" >
     <property name="geometry" >
      <rect>
       <x>15</x>
       <y>92</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Service Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="stopWeb" >
     <property name="geometry" >
      <rect>
       <x>465</x>
       <y>97</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5" >
    <attribute name="title" >
     <string>OPIs</string>
    </attribute>
    <widget class="QPushButton" name="stopOPI" >
     <property name="geometry" >
      <rect>
       <x>467</x>
       <y>99</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbOPISvc" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>120</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>ntpd</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>sysMonitor</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="startOPI" >
     <property name="geometry" >
      <rect>
       <x>369</x>
       <y>99</y>
       <width>90</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13" >
     <property name="geometry" >
      <rect>
       <x>17</x>
       <y>115</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Service Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15" >
     <property name="geometry" >
      <rect>
       <x>17</x>
       <y>55</y>
       <width>137</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Baekmuk Gulim</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Server Name</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbOPI" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>60</y>
       <width>164</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text" >
       <string>opi01</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi02</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi03</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi04</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi05</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi06</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi07</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi08</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi09</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi10</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi11</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi12</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi13</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi14</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi15</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi16</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi17</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi18</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>opi19</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>139</y>
     <width>695</width>
     <height>329</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="rowCount" >
    <number>0</number>
   </property>
   <property name="columnCount" >
    <number>4</number>
   </property>
   <column>
    <property name="text" >
     <string>Server</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Service</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Process Count</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>87</y>
     <width>118</width>
     <height>37</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>all</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>ca</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>mdsplus</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>da</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>web</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>backup</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi01</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi02</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi03</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi04</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi05</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi06</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi07</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi08</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi09</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi10</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi11</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi12</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi13</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi14</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi15</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi16</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi17</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi18</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>opi19</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>226</x>
     <y>88</y>
     <width>102</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text" >
    <string>Chcek Service</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>167</x>
     <y>10</y>
     <width>296</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Baekmuk Gulim</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Service Checker</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14" >
   <property name="geometry" >
    <rect>
     <x>4</x>
     <y>83</y>
     <width>89</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Baekmuk Gulim</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Server</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leresult" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>700</y>
     <width>541</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>700</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Baekmuk Gulim</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Result</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16" >
   <property name="geometry" >
    <rect>
     <x>424</x>
     <y>47</y>
     <width>102</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Baekmuk Gulim</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Prev Check:</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leprechktime" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>51</y>
     <width>165</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17" >
   <property name="geometry" >
    <rect>
     <x>424</x>
     <y>86</y>
     <width>102</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Baekmuk Gulim</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Curr Check:</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lecurchktime" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>90</y>
     <width>165</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
